p/themes/Dark-pink/metadata.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Dark pink",
+  "author": "Miicat_47",
+  "description": "Nice dawk pink theme. >_< UwU",
+  "version": 0.1,
+  "files": ["_template.css", "pinkdark.css"]
```

```suggestion
  "files": ["_template.css", "../Dark/dark.css", "pinkdark.css"]
```

@math-GH Alternative suggestion to copying `dark.css`

---

thanks. I thought about this too, but did not checked if it works. Great.

---

Nice, did that